test.csv

```diff
@@ -0,0 +1,3 @@
+fruit,quantity
+banana,152
+apple,90
```

Comment:
This file was generated during tests so It should be deleted after the test is done